### DISCUSSION TITLE
Open rosbridge connection , can compression only be "cbor-raw"?

### DISCUSSION THREAD

**Discussion by qq418745** [foxglove/community#589] [Q&A] [answered]
I encountered a bug in ROS1.

Similar to this question:  https://github.com/RobotWebTools/rosbridge_suite/issues/610  

Can I can use other compression?  ("connection: none")

**Comment by defunctzombie** [comment, marked as answer]
Compression can only be `cbor-raw` for rosbridge connections from Studio. `cbor-raw` is the compression mode for sending the original ROS serialized message data which is what studio requires.
